vcf/vcf.go

```diff
@@ -14,15 +15,16 @@ import (
 //slices of slice
 type Vcf struct {
 	Chr    string
-	Pos    int64
+	Pos    int
 	Id     string
-	Ref    string
-	Alt    string
+	Ref    []dna.Base
+	Alt    [][]dna.Base
```

Comment:
Something to consider for the 2D dna.Base slice is to instead use a bytes.Buffer and convert the bytes.Buffer into the [][]dna.Base only when necessary because I think in general most SNPs would only occupy one byte and the number of Alt alleles don't extend out too far either.